packages/image_picker/image_picker/android/src/main/java/io/flutter/plugins/imagepicker/ImagePickerDelegate.java

```diff
@@ -368,6 +393,26 @@ private void handleChooseImageResult(int resultCode, Intent data) {
     finishWithSuccess(null);
   }
 
+  private void handleChooseMultiImageResult(int resultCode, Intent data) {
+    if (resultCode == Activity.RESULT_OK && data != null) {
```

Comment:
Are there any other error situations that we need to handle? Is it (resultCode != Activity.RESULT_OK) always because user cancelled the flow?

Reply:
I think we don't have any other situations. Yes.